assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/AnalyticsStats.js

```diff
@@ -85,6 +88,12 @@ const AnalyticsStats = ( {
 		},
 	};
 
+	if ( isZeroReport( data ) ) {
+		options.vAxis.viewWindow.max = 1;
```

Comment:
The `max` here should be set with a more appropriate value to use for the max y axis value. Using a value of `1` results in fractional increments between `0` and `1` which doesn't make sense for Goals because you can't partially complete a goal and it isn't displaying an average.

Similarly, bounce rate is a percentage so it would naturally have a maximum of `100` (also used elsewhere).